.gitignore

```diff
@@ -49,4 +49,4 @@ bower.json
 # Ignore Byebug command history file.
 .byebug_history
 
-.idea/
+.idea/
```

Comment:
unneeded change, revert it

Reply:
![screenshot from 2017-12-27 20-58-13](https://user-images.githubusercontent.com/245831/34390489-b394f612-eb48-11e7-8904-c9f4fbb76d31.png)

Reply:
your change is that u removed last empty line in this file

Reply:
actually if you have often issue with .idea folder I think it could be moved to .gitignore?
I'm just not using idea and don't know what problems you can have with it

Reply:
this change only removes blank new line in the end of file
no need in that really